resources/js/features/root/components/header/FacilityControl.tsx

```diff
@@ -0,0 +1,94 @@
+import { useNavigate, useParams } from "@solidjs/router";
+import { createQuery, useQueryClient } from "@tanstack/solid-query";
+import { FacilityResource, System, User } from "data-access/memo-api";
+import {
+  Component,
+  For,
+  Match,
+  Show,
+  Switch,
+  createEffect,
+  createMemo,
+} from "solid-js";
+import { facilityId, setFacilityId } from "state/facilityId.state";
+
+export const FacilityControl: Component = () => {
+  const navigate = useNavigate();
+  const facilities = useFacilityControl();
+
+  return (
+    <Show when={facilities()}>
+      {(facilities) => (
+        <Switch fallback={null}>
```

Comment:
A co jest domyślnie że to trzeba overridować? A undefined nie działa? Bo brzydko bardzo taki null, próbujmy udawać że nulli nie ma. Może `<></>`, takie jest przyjemne :)

Reply:
null jest poprawny, prawdopodobnie domyślny fallback to właśnie null, nie wiem czemu to dodałem

Reply:
No niestety jest poprawny. Chociaż to nie react, więc można używać też undefined w różnych miejscach, jak Filip ustalił. No ale tutaj pewnie wcale niepotrzebne.

Reply:
wolę zostawić tak, jakoś odrzuca mnie pisanie `<></>` jako "wyświetl nic"